PyDSS/metrics.py

```diff
@@ -237,8 +237,8 @@ def _initialize_containers(self, values):
                 assert prop.name == prop_name, f"{prop.name} {prop_name}"
             if prop.data_conversion != DataConversion.NONE:
                 vals = [
-                    convert_data(x.FullName, prop_name, y, prop.data_conversion)
-                    for x, y in zip(self._dss_objs, values)
+                    convert_data(x, prop_name, y, prop.data_conversion)
+                    for x, y in zip(names, values)
```

Comment:
The order should have already been correct, but this makes it explicit.

Reply:
Hi dan this looks good to me you can go ahead with the merge